orchagent/routeorch.cpp

```diff
@@ -1441,140 +1467,155 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
             return false;
         }
     }
-    else if (nextHops.getSize() == 0)
+    else if (ctx.nhg_index.empty())
```

Comment:
This is too much of code change due to indentation. Please have this as first case in the beginning and retain the existing flow

Reply:
Done